crypto/ec/ec_key.c

```diff
@@ -9,6 +9,7 @@
  */
 
 #include "internal/cryptlib.h"
+#include "internal/bn_int.h"
```

Comment:
I have seen that this change will also be needed in other keygen (I cant remember if it was in RSA or DH/DSA).

Reply:
If so, should this be a new public function rather than an internal one? If we need it to be FIPS compliant then most probably it's something that external ENGINEs need as well (and should go in a separate PR).